scripts/check-redact-log

```diff
@@ -7,12 +7,12 @@ function error_msg() {
 }
 
 if [[ "$(uname)" == "Darwin" ]] ; then
-   if grep -r -n --color=always --include '*.rs' --exclude hex.rs --exclude-dir target 'encode_upper' . | grep -v log_wrappers ; then
+   if grep -r -n --color=always --include '*.rs' --exclude hex.rs --exclude tikv-ctl.rs --exclude-dir target 'encode_upper' . | grep -v log_wrappers ; then
```

Comment:
@skyzh I disabled the check on tikv-ctl, is it ok?

Reply:
Use log_wrappers::hex_encode_upper instead. This check is used to notify developers not to directly print out hex value of user data to console, so don't change it.

Reply:
tikv-ctl should be an exexception because it doesn't read the config toml and also will not be used in daily work.

Reply:
it doesn't relate to config toml. We should ensure tikv-ctl outputs the key or value in hex format.